src/Parameters.cpp

```diff
@@ -60,6 +61,7 @@ bool cfg_tune_only;
 float cfg_puct;
 float cfg_softmax_temp;
 float cfg_fpu_reduction;
+float cfg_root_temp;
```

Comment:
It looks like we don't change this value anywhere?  So could just inline the 1.0 in the one place it's used.